test/list.go

```diff
@@ -160,6 +160,24 @@ func DescribeListTestsFor(ctx *common.TestContext, t TestCase) bool {
 				expectedStatusCode:          http.StatusOK,
 			},
 		),
+		Entry("returns 200 for greater than or equal queries",
```

Comment:
write negative tests that validate the status code is 400 when the input is invalid